### ISSUE TITLE
NPC: Baron Rivendare

### ISSUE THREAD

**Issue by underlord-oberon** [plantain-gnocchis/wow-voiceover-tbc-wotlk#74]
Add-on version: AI_VoiceOver_v1.4.5
Data version: AI_VoiceOverData_VanillaExtra_v1.4

Plaguelands: The Scarlet Enclave / Crypt of Remembrance  / NPC: Baron Rivendare
Link: https://www.wowhead.com/wotlk/npc=28910/baron-rivendare

Greeting: the voice-over suddenly silences during playback.

**Comment by carevass** [comment]
This should be fixed in AI_VoiceOverData_WoTLK_v1.1.
